### PR TITLE
Sv differ fixes

### PR DESCRIPTION
**Pull request by chejennifer**
- update to_vars to use a non-deprecated endpoint
- add nodejs queries that can be used by sv differ tool
- update reading catalog for sv differ

### CODE REVIEW

File: nl_server/config_reader.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def merge_vertex_ai_configs(m: ModelConfig,
   return VertexAIModelConfig(**merged)
 
 
-def read_catalog() -> Catalog:
+def read_catalog(catalog_dict: Dict = None) -> Catalog:
```
Comment (shifucun) [inline nl_server/config_reader.py:101]: I like this change and we should eventually make this 

`read_catalog(catalog_paths: [] = DEFAULT_PATHS, catalog_dict: Dict = None)`

Right now the custom DC path prevents the path to be constant, and really that should be passed in as the catalog_dict string value. 

Can you add a TODO for this to remind me of the change.

**Review by shifucun — APPROVED** [review]
nice update!

File: nl_server/config_reader.py (hunk 1 above)
Comment (chejennifer) [reply nl_server/config_reader.py:104]: added TODO

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**